contracts/abstracts/Controlable.sol

```diff
@@ -80,6 +80,24 @@ abstract contract Controlable is AccessControlUpgradeable {
     }
   }
 
+  function _isBlacklistedUser(address userAddress) internal view returns (bool isBlacklisted) {
+    if (blacklistUser[userAddress] == true) {
+      return true;
+    }
+  }
+
+  function _isBlacklistedToken(address tokenContract, uint256 tokenId) internal view returns (bool isBlacklisted) {
+    if (blacklistToken[tokenContract][tokenId] == true) {
+      return true;
+    }
```

Comment:
blacklistToken[tokenContract][tokenId] already equal bool
so remove the if and just add
```solidity
return blacklistToken[tokenContract][tokenId];
```